decidim-core/app/queries/decidim/organization_participatory_processes.rb

```
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+module Decidim
+  # This query class returns the visible Participatory Processes given an
+  # Organization.
+  class OrganizationParticipatoryProcesses < Rectify::Query
+    def initialize(organization)
+      @organization = organization
+    end
+
+    def query
+      raise MandatoryOrganization unless @organization
```

Why raise instead of just returning `nil`?